src/commands/deleteCosmosDBAccount.ts

```diff
@@ -21,7 +22,9 @@ export async function deleteCosmosDBAccount(node: AzureTreeItem): Promise<void>
             await client.databaseAccounts.deleteMethod(resourceGroup, accountName);
         });
         // don't wait
```

Comment:
nit: it makes sense to keep this "don't wait" by the `showInformationMessage` call if we even need to keep it at all

Reply:
might not need it actually